compiler-core/src/manifest.rs

```diff
@@ -193,7 +193,7 @@ pub enum ManifestPackageSource {
     Local { path: Utf8PathBuf }, // should be the canonical path
 }
 
-fn ordered_map<S, K, V>(value: &HashMap<K, V>, serializer: S) -> Result<S::Ok, S::Error>
+pub fn ordered_map<S, K, V>(value: &HashMap<K, V>, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
If this is public it should not be in the manifest module as it's unrelated to the manifest.

Reply:
What would be a good place to put this function?

Reply:
It's annoyingly generic isn't it? Perhaps `io` at a push